packages/password-input/src/TogglePassword/TogglePassword.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { VisuallyHidden } from '@leafygreen-ui/a11y';
+import { cx } from '@leafygreen-ui/emotion';
+import Icon from '@leafygreen-ui/icon';
+import IconButton from '@leafygreen-ui/icon-button';
+
+import { baseStyles, sizeStyles } from './TogglePassword.styles';
+import { TogglePasswordProps } from './TogglePassword.types';
+
+export const TogglePassword = ({
```

Comment:
super nit: I feel like `PasswordToggle` is a more intuitive component name

Reply:
i agree